### PR TITLE
add avatar

### PR DESCRIPTION
**Pull request by tuansaker1412**
------------------------------------------
[o] Build Success!
------------------------------------------

[o] rubocop: success
[o] rspec: success
[o] rails_best_practices: success
[o] bundle-audit: success
[o] eslint: success
[o] scss-lint: success
[o] brakeman: success
[o] reek: success

### CODE REVIEW

**Comment by tuansaker1412** [comment]
ready

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -11,6 +11,6 @@ def configure_permitted_parameters
   end
 
   def update_account
-    devise_parameter_sanitizer.permit(:account_update, keys: [:name])
+    devise_parameter_sanitizer.permit(:account_update, keys: [:name, :avatar])
```
Comment (huongnguyenmta) [inline app/controllers/application_controller.rb]: remove ()

**Comment by huongnguyenmta** [comment]
commented

**Comment by tuansaker1412** [comment]
ready

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -7,10 +7,10 @@ class ApplicationController < ActionController::Base
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+    devise_parameter_sanitizer.permit :sign_up, keys: [:name]
```
Comment (huongnguyenmta) [inline app/controllers/application_controller.rb:10]: `keys: :name`

**Comment by huongnguyenmta** [comment]
commented-fix nốt tí tẹo chế MỢT choa =))-no

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (tuansaker1412) [reply app/controllers/application_controller.rb:10]: cái này bỏ là lỗi chị ơi ._.

Comment (huongnguyenmta) [reply app/controllers/application_controller.rb:10]: Ok nhé, chị check lại rồi :D
`keys: - An Array of keys that also should be permitted`

**Comment by tuansaker1412** [comment]
ready

File: app/controllers/application_controller.rb (hunk 2 above)
Comment (tuansaker1412) [reply app/controllers/application_controller.rb:10]: <3

**Pull request merged**